tests/author_test.py

```diff
@@ -0,0 +1,95 @@
+"""Author Tests"""
+
+import unittest
+from app import create_app
+from database.models import Author
+
+
+class AuthorTest(unittest.TestCase):
+    """Author Test Class"""
+
+    def setUp(self):
```

Comment:
```suggestion
    def set_up(self):
```

Reply:
This leads to an error. unittest uses "setUp" method to setup for each test case.